libs/pika/command_line_handling/tests/unit/CMakeLists.txt

```diff
@@ -1,6 +1,27 @@
-# Copyright (c) 2020 The STE||AR-Group
-#               2011 Bryce Adelstein-Lelbach
+# Copyright (c) 2021 The STE||AR-Group
```

Comment:
Just add "ETH Zurich"?

Reply:
wtf? copy paste madeness.

Reply:
Did you forget to push this change?

Reply:
no... are you somehow getting the diffs reversed? it looks fixed to me

Reply:
hmmm wrong ffile. sorry.